Projects/Foundations/Schemas/Docs-Tutorials/D/Developers/Developers-Intro/developers-intro-001-tutorial-intro-for-developers.json

```diff
@@ -10,6 +10,11 @@
         "text": "This tutorial walks you through the technical aspects of Superalgos so that you may take advantage of the infrastructure in place as efficiently as possible.",
         "translations": [
             {
+                "language": "DE",
+                "text": "Dieses Tutorial führt Sie durch die technischen Aspekte von Superalgos, so dass Sie die Vorteile der vorhandenen Infrastruktur so effizient wie möglich nutzen können.",
+                "updated": 1641729113684
+                },
+                {
```

Comment:
It would be great if you could align both lines more to the left. You can orient on the part down below how far to the left.(You can do it here on github by clicking on the file and then edit file)